lib/lists.nix

```diff
@@ -35,13 +40,25 @@ rec {
         else op (elemAt list n) (fold' (n + 1));
     in fold' 0;
 
-  /* Left fold: `fold op nul [x_1 x_2 ... x_n] == op (... (op (op nul
-     x_1) x_2) ... x_n)'.
+  /* fold is an alias of foldr for historic reasons */
+  # FIXME(Profpatsch): deprecate?
+  fold = foldr;
```

Comment:
If I don't know the difference and implications between foldl and foldr, which one should I use by default?